lib/doofinder_api_items.php

```diff
@@ -55,6 +60,24 @@ public function deleteBulk($payload)
         return $this->post($url, $payload);
     }
 
+    /**
+     * Make a request to the API to reprocess all the feeds
+     *
+     * @param array items ids
+     */
+    public function invokeReindexing()
+    {
+        $installation_id = Configuration::get('DF_INSTALLATION_ID');
+        $json_data = json_encode(['query' => 'mutation { process_store_feeds(id: "' . $installation_id . '", callback_url: "' . $this->getProcessCallbackUrl() . '") { id }}']);
```

Comment:
installation_id must be a parameter of the function.
Also callback_url

Reply:
Changed on: https://github.com/doofinder/doofinder-prestashop/pull/143/commits/13a4e4631a3fff59967320600ca685099550cd59